tests/base.py

```diff
@@ -239,5 +242,17 @@ def __init__(self):
         self.template_system = self
         self.name = 'mako'
 
+    def _activate_plugins_of_category(self, category):
```

Comment:
This is code duplication, is it possible to import `nikola.nikola` and do `FakeSite._activate_plugins_of_category = nikola.nikola.Nikola._activate_plugins_of_category` instead?